Dockerfile

```diff
@@ -3,13 +3,16 @@
 FROM ubuntu:20.04
 
 LABEL maintainer="bharadwajr07"
+WORKDIR /app
+
+# avoid stuck build due to user prompt
+ARG DEBIAN_FRONTEND=noninteractive
 
 RUN apt-get update -y && \
-    apt-get install -y python-pip python-dev
+    apt-get install -y python3.9 python3.9-dev python3-pip python3-wheel build-essential
```

Comment:
may be able to prune this down to the essentials

Reply:
Got it.